Signal/translations/en.lproj/Localizable.strings

```diff
@@ -1538,7 +1538,7 @@
 "SCAN_QR_CODE_VIEW_TITLE" = "Scan QR Code";
 
 /* Indicates a delay of zero seconds, and that 'screen lock activity' will timeout immediately. */
-"SCREEN_LOCK_ACTIVITY_TIMEOUT_NONE" = "None";
+"SCREEN_LOCK_ACTIVITY_TIMEOUT_NONE" = "Instant";
```

Comment:
TIOLI: "Immediately" ?

Reply:
I think Josh said in standup that he thought Instant was best.